lib/core/src/lib/viewer/components/viewer.component.html

```diff
@@ -23,9 +23,9 @@
                         </button>
                     </ng-container>
 
-                    <button *ngIf="allowGoBack"
+                    <button *ngIf="allowGoBack && closeButtonPosition === 'left'"
```

Comment:
Should be `CloseButtonPosition.Left` instead of `left`

Reply:
done